automated install/basic-install.sh

```diff
@@ -1128,8 +1128,10 @@ chooseBlocklists() {
         appendToListsFile "${choice}"
     done
     # Create an empty adList file with appropriate permissions.
-    touch "${adlistFile}"
-    chmod 644 "${adlistFile}"
+    if [ ! -f "${adlistFile}" ]; then
+        touch "${adlistFile}"
+        chmod 644 "${adlistFile}"
```

Comment:
Do we need the `chmod` guarded for the buster thing as well? Otherwise, taking it out of the if statement would correct and pre-existing file with wrong permissions.

Reply:
Yep, that's what I'm doing now.

Reply:
Sorry, I thought you were referring to the `touch` problem, that was what I was fixing.

I see what you are saying.

Reply:
I think he means:

```sh
    if [ ! -f "${adlistFile}" ]; then
        install -m 644 /dev/null "${adlistFile}"
    fi
    chmod 644 "${adlistFile}"
```

Reply:
Yea, but that duplicates the mode changes. We don't need to set 644 right after we created the file with 644.

Reply:
Yeah... I saw that.